### PR TITLE
Fix offline Swagger UI for Litestar

### PR DESCRIPTION
**Pull request by vrslev**

### CODE REVIEW

**Comment by insani7y** [comment]
I would not enforce users to use our hardcoded version of openapi schema rendering, cause in our litestar offline docs package (as well as in fastapi's package) static files are built in to the package.

**Comment by vrslev** [comment]
Meaning, you’d prefer it to be disabled by default? Or I didn’t understand what you mean exactly

**Comment by vrslev** [comment]
> static files are built in to the package

The package with static files is shipped either way

File: microbootstrap/bootstrappers/litestar.py [hunk 1]
```diff
@@ -51,24 +52,28 @@ def bootstrap(self) -> None:
...
+                css_url=f"{self.instrument_config.service_static_path}/swagger-ui.css",
+                standalone_preset_js_url=(
                     f"{self.instrument_config.service_static_path}/swagger-ui-standalone-preset.js"
-                )
-                swagger_bundle_path: str = f"{self.instrument_config.service_static_path}/swagger-ui-bundle.js"
-                swagger_css_url: str = f"{self.instrument_config.service_static_path}/swagger-ui.css"
+                ),
+            )
+            if self.instrument_config.swagger_offline_docs
+            else SwaggerRenderPlugin()
```
Comment (vrslev) [inline microbootstrap/bootstrappers/litestar.py:64]: Or you're telling that we should not select render plugin if offline docs flag is set to `False`?

**Comment by vrslev** [comment]
I've disabled offline docs by default. Also Swagger UI is not enforced if offline docs are disabled.

The fix still applies: otherwise offline docs do not work.

**Pull request merged**